com.unity.render-pipelines.universal/Editor/EditorUtils.cs

```diff
@@ -25,46 +26,202 @@ static class EditorUtils
         public const int lwrpAssetCreateMenuPriorityGroup2 = CoreUtils.assetCreateMenuPriority1 + 50;
         public const int lwrpAssetCreateMenuPriorityGroup3 = lwrpAssetCreateMenuPriorityGroup2 + 50;
 
+        internal enum Unit { Metric, Percent }
+
         internal class Styles
         {
             //Measurements
             public static float defaultLineSpace = EditorGUIUtility.singleLineHeight + EditorGUIUtility.standardVerticalSpacing;
-            public static float defaultIndentWidth = 12;
         }
 
-        public static void DrawCascadeSplitGUI<T>(ref SerializedProperty shadowCascadeSplit)
+        public static void DrawCascadeSplitGUI<T>(ref SerializedProperty shadowCascadeSplit, float distance, int cascadeCount, Unit unit)
```

Comment:
I think this is going to cause API validation to fail as you are changing public function

Reply:
Added original function to deprecated and made it obsolete